manifests/client/config.pp

```diff
@@ -1,5 +1,9 @@
 # private class
-class mcollective::client::config {
+class mcollective::client::config (
+    $owner = 'root',
```

Comment:
Maybe use the ```String``` datatype on all of these?

Reply:
Yes, of course